client/src/modules/stock/lots-duplicates/duplicate-lots.js

```diff
@@ -0,0 +1,165 @@
+angular.module('bhima.controllers')
+  .controller('DuplicateLotsController', DuplicateLotsController);
+
+DuplicateLotsController.$inject = [
+  'LotService', 'NotifyService', 'uiGridConstants', 'StockModalService',
+  'SessionService', 'GridStateService', '$state', 'GridColumnService',
+];
+
+/**
+ * Stock Inventory Controller
+ * This module is a registry page for stock inventories
```

Comment:
These comments should probably be updated to reflect the true nature of the controller.

Reply:
Updated.